src/ontology/mondo-edit.obo

```diff
@@ -543352,6 +543351,7 @@ xref: GARD:16754 {source="MONDO:GARD"}
 xref: icd11.foundation:1736273667 {source="MONDO:equivalentTo"}
 xref: Orphanet:85447 {source="MONDO:equivalentTo"}
 is_a: MONDO:0007100 {source="Orphanet:85447"} ! familial amyloid neuropathy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12405 {source="https://orcid.org/0000-0002-0587-4693"} ! TTR
```

Comment:
the source for this relation should be the orphanet xref since this is where we found the information about the variation.

Reply:
Comment addressed